README.md

```diff
@@ -88,9 +79,186 @@ inputs:
+chimMultimapScoreRange: { type: 'int?', default: 1, doc: "the score range for multi-mapping chimeras below the best chimeric \
+score. Only works with chimMultimapNmax > 1. SF recommends 3" }
+chimNonchimScoreDropMin: { type: 'int?', default: 20,
+doc: "int>=0: to trigger chimeric detection, the drop in the best non-chimeric \
+alignment score with respect to the read length has to be greater than this value. SF recommends 10" }
+chimOutJunctionFormat: { type: 'int?', default: 1, doc: "formatting type for the Chimeric.out.junction file, value 1 REQUIRED for SF" }
+chimOutType: { type: [ 'null', {type: enum, name: chimOutType, symbols: [
+    "Junctions SeparateSAMold WithinBAM SoftClip",
+    "Junctions", "SeparateSAMold",
+    "WithinBAM SoftClip",
+    "WithinBAM HardClip",
+    "Junctions SeparateSAMold",
+    "Junctions WithinBAM SoftClip",
+    "Junctions WithinBAM HardClip",
+    "Junctions SeparateSAMold WithinBAM HardClip",
+    "SeparateSAMold WithinBAM SoftClip",
+    "SeparateSAMold WithinBAM HardClip"
+    ]}],
+default: "Junctions SeparateSAMold WithinBAM SoftClip",
+doc: "type of chimeric output. Args are additive, and defined as such - Junctions: Chimeric.out.junction. SeparateSAMold: output old SAM into separate Chimeric.out.sam file \
+WithinBAM: output into main aligned BAM files (Aligned.*.bam). WithinBAM HardClip: hard-clipping in the CIGAR for supplemental chimeric alignments \
+WithinBAM SoftClip:soft-clipping in the CIGAR for supplemental chimeric alignments" }
+chimScoreDropMax: { type: 'int?', default: 20,
+doc: "max drop (difference) of chimeric score (the sum of scores of all chimeric segments) from the read length. AR recommends 30" }
+chimScoreJunctionNonGTAG: { type: 'int?', default: -1, doc: "penalty for a non-GT/AG chimeric junction. \
+default -1, SF recommends -4, AR -1" }
+chimScoreSeparation: { type: 'int?', default: 10,
+doc: "int>=0: minimum difference (separation) between the best chimeric score and the next one. AR recommends 1" }
+chimSegmentMin: { type: 'int?', default: 12, doc: "minimum length of chimeric segment length, if ==0, no chimeric output. \
+REQUIRED for SF, 12 is their default, AR recommends 10" }
+chimSegmentReadGapMax: { type: 'int?', default: 0, doc: "maximum gap in the read sequence between chimeric segments. AR recommends 3" }
+outFilterMultimapNmax: { type: 'int?', default: 20, doc: "max number of multiple alignments allowed for \
+a read: if exceeded, the read is considered unmapped. ENCODE value is default. AR recommends 50" }
+peOverlapMMp: { type: 'float?', default: 0.01, doc: "maximum proportion of mismatched bases in the overlap area. SF recommends 0.1" }
+peOverlapNbasesMin: { type: 'int?', default: 0,
+doc: "minimum number of overlap bases to trigger mates merging and realignment. Specify >0 value to switch \
+on the 'merging of overlapping mates'algorithm. SF recommends 12,  AR recommends 10" }
```

Comment:
For this whole block I feel like we should use the more readable multiline yaml format
```suggestion
peOverlapNbasesMin:
  type: int?
  default: 0
  doc: >-
    minimum number of overlap bases to trigger mates merging and realignment.
    Specify >0 value to switch on the 'merging of overlapping mates'algorithm.
    SF recommends 12,  AR recommends 10
```

https://onlineyamltools.com/prettify-yaml